python/tidy/servo_tidy/tidy.py

```diff
@@ -147,13 +143,47 @@ def filter_files(start_dir, only_changed_files, progress):
         yield file_name
 
 
+def uncomment(line):
+    for c in COMMENTS:
+        if line.startswith(c):
+            if line.endswith("*/"):
+                return line[len(c):(len(line) - 3)].strip()
+            return line[len(c):].strip()
+
+
+def licensed_mpl(header):
+    return licenses.MPL.replace('\n', ' ') in header
```

Comment:
Instead of doing a string replacement for every function call, we could just have a global variable like `mpl = MPL.replace('\n', ' ')` or something at the top.